### ISSUE TITLE
WooCommerce Analytics: remove module from the Jetpack plugin and stop requiring the package.

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#38295]
Once the feature is included in the WooCommerce plugin, it can safely be removed from the Jetpack plugin.

@puntope When do you think we could reasonably stop loading the package in the Jetpack plugin? We release a new version of the Jetpack plugin monthly. When can we assume a version of the WooCommerce plugin will have been out and available long enough so most site owners have updated and we can remove the feature from Jetpack?

******

Epic: #35511

**Comment by puntope** [comment]
Hey @jeherve thanks for the ping. 

That depends on your compatibility policy. I created a PR which is already merged to avoid initialize the plugin if it's loaded in WooCommerce. I assume that it will be be released in Jetpack on 6th August as Jetpack team confirmed to me recently. 

However, it can be the case that users are using the new version of Jetpack but an old version of WooCommerce (without the package loading in Woo) in those cases Jetpaxck is still initializing the package.

In our plugins (like GLA) we offer L-2 policy. But not sure about Jetpack. Let's assume it is also L-2 policy. Then let's assume also that the feature is released on WooCommerce v9.4 (I can confirm this soon). Then you should keep the package in Jetpack minimum until WooCommerce v9.6 and delete it in v9.7.  

Does this make sense?

**Comment by jeherve** [comment]
> let's assume also that the feature is released on WooCommerce v9.4 (I can confirm this soon). Then you should keep the package in Jetpack minimum until WooCommerce v9.6 and delete it in v9.7.

Yep, this is what I was looking for, thank you! Do you already know when those versions of Woo will be released, approximately?

**Comment by puntope** [comment]
> Yep, this is what I was looking for, thank you! Do you already know when those versions of Woo will be released, approximately?

The versions were just examples. We need to discuss how we want to tackle the tracking permissions (GDPR etc) Once this is clarified I can commit to a Woo version and also I can ping JP team if that's ok.

**Comment by jeherve** [comment]
Sounds good, thank you!

**Comment by chrisbliss18** [comment]
@puntope Has there been any progress on the tracking permissions discussions? Are the specific Woo versions now known?

**Comment by puntope** [comment]
Hey @chrisbliss18 the project is currently on hold due GDPR issues. See progress here
pcShBQ-1Vt-p2
